packages/atomic/src/utils/local-storage-utils.spec.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  buildSearchEngine,
+  getSampleSearchEngineConfiguration,
+} from '@coveo/headless';
+import {SafeStorage, StorageItems} from './local-storage-utils';
+
+describe('Safe local storage', () => {
+  let storage: SafeStorage;
+
+  beforeEach(() => {
+    const engine = buildSearchEngine({
+      configuration: getSampleSearchEngineConfiguration(),
+    });
+    storage = new SafeStorage(engine.logger);
```

Comment:
If `SafeStorage` accepts a [mock storage object](https://github.com/coveo/ui-kit/pull/1524/files#diff-8b9f1b4527bd3c95694c0e9f98c1f1f022e7dccf5663aa30912c3e1a88254a69R17), it would be possible to remove the global `jest-localstorage-mock` config and dependency.

As you prefer,

Reply:
Given a choice, I prefer to not pollute `SafeStorage` implementation with something that is only needed for testing, and to "pollute" the testing setup.

Because IMO, `SafeStorage` implementation is more important than the testing setup, since that's what we actually ship in the product.